index.html

```diff
@@ -791,7 +792,196 @@ <h2>Disabled and Read Only input</h2>
+                    </ul>
+                </div>
+                <pre>
+                    <code class="html">
+&lt;ul class="list ul-disc"&gt;
+    &lt;li&gt;list item 1&lt;/li&gt;
+    &lt;li&gt;list item 2&lt;/li&gt;
+    &lt;li&gt;list item 3&lt;/li&gt;
+    &lt;li&gt;list item 4&lt;/li&gt;
+&lt;/ul&gt;
+&lt;ul class="list ul-square"&gt;
+    &lt;li&gt;list item 1&lt;/li&gt;
+    &lt;li&gt;list item 2&lt;/li&gt;
+    &lt;li&gt;list item 3&lt;/li&gt;
+    &lt;li&gt;list item 4&lt;/li&gt;
+&lt;/ul&gt;
+&lt;ul class="list ul-circle"&gt;
+    &lt;li&gt;list item 1&lt;/li&gt;
+    &lt;li&gt;list item 2&lt;/li&gt;
+    &lt;li&gt;list item 3&lt;/li&gt;
+    &lt;li&gt;list item 4&lt;/li&gt;
+&lt;/ul&gt;
+                    </code>
+                </pre>
+
+                <h2>Ordered List with bullets</h2>
+                <div class="comps-container">
+                    <ol class="list ol-number">
+                        <li>Numbered list item</li>
+                        <li>Numbered list item</li>
+                        <li>Numbered list item</li>
+                        <li>Numbered list item</li>
+                    </ol>
+                    <ol class="list ol-lower-alpha">
+                        <li>List item with lowercase alphabets</li>
+                        <li>List item with lowercase alphabets</li>
+                        <li>List item with lowercase alphabets</li>
+                        <li>List item with lowercase alphabets</li>
+                    </ol>
+                    <ol class="list ol-upper-alpha">
+                        <li>List item with uppercase alphabets</li>
+                        <li>List item with uppercase alphabets</li>
+                        <li>List item with uppercase alphabets</li>
+                        <li>List item with uppercase alphabets</li>
+                    </ol>
+                    <ol class="list ol-lower-roman">
+                        <li>List item with lowercase roman</li>
+                        <li>List item with lowercase roman</li>
+                        <li>List item with lowercase roman</li>
+                        <li>List item with lowercase roman</li>
+                    </ol>
+                    <ol class="list ol-upper-roman">
+                        <li>List item with uppercase roman</li>
+                        <li>List item with uppercase roman</li>
+                        <li>List item with uppercase roman</li>
+                        <li>List item with uppercase roman</li>
+                    </ol>
+                    <ol class="list ol-number" reversed>
+                        <li>Reversed list item</li>
+                        <li>Reversed list item</li>
+                        <li>Reversed list item</li>
+                        <li>Reversed list item</li>
+                    </ol>
+                </div>
+                <pre>
+                    <code class="html">
+&lt;ol class="list ol-number"&gt;
```

Comment:
indent it properly. To remove the white spaces from the code block you can style the code tag either by giving white-space: pre-line/pre-wrap or by giving negative margin.

Reply:
Thank you @chanchal16. I wasn't aware of this propery. Will make the change